pkg/agent/host_configuration_windows.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2020 Antrea Authors
```

Comment:
As they just holds different functions called in agent.go, should they just be called agent_linux.go and agent_windows.go? I suggest this because I saw all XX_OS.go are following the convention, though I don't find an official style guide.